ko/README.md

```diff
@@ -14,7 +14,7 @@
     [이수진](https://github.com/sujinleeme)
 > - 3차 : 2017. 5. 19.
     [이수진](https://github.com/sujinleeme)
-> - 4차 : 2019. 2. 5. [이재열](https://github.com/malkoG)
+> - 4차 : 2019. 2. 5. [이재열](https://github.com/malkoG), [정수민](https://github.com/soomin-jeong)
```

Comment:
It would be a trivial stuff. Would you break a line?

Reply:
Oh, I've done. @sujinleeme